common/lcs/tests/serde.rs

```diff
@@ -281,15 +320,26 @@ fn sequence_too_long() {
     }
 }
 
+#[test]
+fn variable_lengths() {
+    assert_eq!(to_bytes(&vec![(); 1]).unwrap(), vec![0x01]);
+    assert_eq!(to_bytes(&vec![(); 128]).unwrap(), vec![0x80, 0x01]);
+    assert_eq!(to_bytes(&vec![(); 255]).unwrap(), vec![0xff, 0x01]);
```

Comment:
Hmmm... little endian isn't fun... this took me way to long to figure out...

Reply:
but now you are a master at ULEB128!